arch/x86/x86_64/include/uk/asm/ctx.h

```diff
@@ -31,6 +31,15 @@
 #error Do not include this header directly
 #endif
 
+/* 512 for the actual structure + 64 bytes for the XSAVE header */
+#define UKARCH_ECTX_SAVE_MAX_SIZE			576
```

Comment:
Although ectx size can depend on cpu features in x86, that's not necessarily true for other architectures, so perhaps it would be nicer to use a unified name i.e. `UKARCH_ECTX_SIZE` (which aligns nicely with `UKARCH_STACK_SIZE`) and leave a comment in the case of x86_64 simiar to:
```
#define UKARCH_ECTX_SIZE    576 /* Max possible size */
```